tests/test_fx.py

```diff
@@ -456,5 +458,12 @@ def test_normalize():
     close_all_clips(locals())
 
 
+def test_normalize_muted():
+    make_frame = lambda t: np.array([0.0])
+    clip = AudioClip(make_frame, duration=1, fps=44100)
+    clip = audio_normalize(clip)
```

Comment:
To make sure, could you check that the returned array is properly muted? Something like this:
```suggestion
    z_array = np.array([0.0])
    make_frame = lambda t: z_array
    clip = AudioClip(make_frame, duration=1, fps=44100)
    clip = audio_normalize(clip)
    assert np.array_equal(clip.to_soundarray(), z_array)
```

Reply:
Good idea